### PR TITLE
resource/alicloud_ess_scaling_configuration: add attribute custom_priorities.

### PR DESCRIPTION
**Pull request by fuliu-zln**
…orities.

### CODE REVIEW

File: website/docs/r/ess_scaling_configuration.html.markdown [hunk 1]
```diff
@@ -149,6 +149,7 @@ The following arguments are supported:
...
 * `network_interfaces` - (Optional, Available since v1.235.0) Specify NetworkInterfaces.N to configure primary and secondary ENIs. In this case, specify at least one primary ENI. If you set NetworkInterfaces.N.InstanceType to Primary, a primary ENI is configured. If you set NetworkInterfaces.N.InstanceType to Secondary or leave the parameter empty, a secondary ENI is configured. See [`network_interfaces`](#network_interfaces) below for details.
+* `custom_priorities` - (Optional, Available since v1.236.0) You can use CustomPriorities to specify the priority of a custom ECS instance type + vSwitch combination. See [`custom_priorities`](#custom_priorities) below for details.
```
Comment (ChenHanZhang) [inline website/docs/r/ess_scaling_configuration.html.markdown:152]: 最新版本1.238.0

Comment (fuliu-zln) [reply website/docs/r/ess_scaling_configuration.html.markdown]: done

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**